sklearn/svm/_classes.py

```diff
@@ -843,7 +843,7 @@ class SVC(BaseSVC):
     >>> print(clf.predict([[-0.8, -1]]))
     [1]
 
-    For a comaprison of the SVC with other classifiers see:
+    For a comparison of the SVC with other classifiers see:
     :ref:`sphx_glr_auto_examples_classification_plot_classification_probability.py`.
```

Comment:
Nice catch! I just noticed there are a few places in the documentation where this particular typo appears. Probably result from copy-pasting from already merged pulled request. Would you feel like fixing those?

Being said that, I don't think the intention of the linked example was to compare `SVC` with other classifiers as stated here, but rather to showcase the use of `DecisionBoundaryDisplay` when plotting `predict_proba` in multiclass classification. I then suggest to completely remove this paragraph, as one day we may use a different set of estimators to serve the real purpose.

Reply:
Thinking about it, there is no need to remove the paragraph. It can be rephrased to something more inline with:
```suggestion
    For an example on how to enable probability estimates for SVC, see:
    :ref:`sphx_glr_auto_examples_classification_plot_classification_probability.py`.
```